src/test/java/scenarios/hooks/OrderScenarioHooks.java

```diff
@@ -54,23 +60,23 @@ public void createOrder() throws JsonProcessingException {
         order.setAccountId(accountId);
         order.setEffectiveDate(date);
         order.setStatus("Draft");
-        ApiResponse apiResponse = ApiRequestManager.create(Endpoint.ORDERS, pathParams, order);
+        ApiResponse apiResponse = ApiRequestManager.create(Endpoints.ORDERS.getEndpoint(), pathParams, order);
         orderId = apiResponse.getBody(CreatedResponse.class).getId();
     }
 
     @After(value = "@GetOrder or @UpdateOrder or @CreateOrder")
     public void setDownOrder() {
         Map<String,String> pathParams = new HashMap<>();
-        pathParams.put(ElementParam.ID,orderId);
-        ApiResponse apiResponse = ApiRequestManager.delete(Endpoint.ORDER, pathParams);
+        pathParams.put(Endpoints.ID.getEndpoint(),orderId);
```

Comment:
Seems a little bit redundant. making use of the enum may be making it somewhat more difficult to read.